libraries/abstractions/transport/utest/include/transport_interface.h

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
Please delete this and use the transport_interface.h from the coreMQTT submodule.  We are making a strong effort to use not have more copies.
Please see :
https://github.com/aws/amazon-freertos/blob/master/libraries/abstractions/transport/CMakeLists.txt#L6

Reply:
Will change it, thanks

Reply:
fixed in 5c6afe0c590c76d8c90a6972c9798058702963f1

resolving this comment, feel free to let me know if need more work.